index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>LAB-2</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/topnav.css" />
+    <link rel="stylesheet" href="css/sidenav.css" />
+    <link rel="stylesheet" href="css/nav-btn.css" />
+    <link rel="stylesheet" href="css/menu.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"/>
+  </head>
+  <body>
+    <nav class="menu-btn" id="main-menu" onclick="clickMenu()">
+      <i class="fas fa-bars"></i>
+    </nav>
+    <nav class="sidenav" id="side-nav">
+        <ul class="nav-btns">
+          <li>
+              <a href="#">PSD WEBSITES</a> 
+          </li>
+          <li>
+              <a href="#">UI KITS</a>
+          </li>
+          <li>
+              <a href="#">PREMIUM PSD FILES</a>
+          </li>
+          <li>
+              <a href="#">ABOUT</a>
+          </li>
+          <li>
+              <a href="#">BLOG</a>
+          </li>
+        </ul>
+    </nav>
+    <nav class="topnav">
+      <h1>Navigation</h1>
+        <ul class="nav-btns">
+          <li>
+              <a href="#">PSD WEBSITES</a>
+          </li>
+          <li>
+              <a href="#">UI KITS</a>
+          </li>
+          <li>
+              <a href="#">PREMIUM PSD FILES</a>
+          </li>
+          <li>
+              <a href="#">ABOUT</a>
+          </li>
+          <li>
+              <a href="#">BLOG</a>
+          </li>
+        </ul>
+    </nav>
+    </div>
+    <a href="http://blazrobar.com/" class="banner">
+      <section class="banner-content">
+        <h1>Awesome Free PSD<br/>templates & Resources</h1>
+        <h2 class="link-text">
+          www.BlazRobar.com
+          <i class="fas fa-angle-right"></i>
+        </h2>
+      </section>
+    </a>
+    <section class="link-box">
+      <h2 class="link-box-text">
+        I'm like the Robin Hood of PSD files.
+      </h2>
+      <a href="http://blazrobar.com/" class="arrow-box">
+        <i class="fas fa-angle-right"></i>
+      </a>
+    </section>
+    <section class="media">
+      <a href="http://blazrobar.com/">
+        <img src="img/1.png" class="item"/>
+      </a>
+      <img src="img/2.png" class="item" />
+      <img src="img/3.png" class="item" />
+    </section>
+    <article>
+      <h1>BlazRobar.com</h1>
+      <div class="content">
+        <span>
+          <p>
```

Comment:
Do not place block elements inside inline elements please, try to use some other tags instead of span